ocsci/tests/test_config.py

```diff
@@ -11,10 +11,68 @@ def reset_config(self):
         ocsci.config.reset()
 
     def test_defaults(self):
-        ocsci.main.init_ocsci_conf()
-        env_data = getattr(ocsci.config, 'ENV_DATA')
-        reporting_data = getattr(ocsci.config, 'REPORTING')
-        run_data = getattr(ocsci.config, 'RUN')
-        assert env_data['rook_image'] == 'rook/ceph:master'
-        assert reporting_data['email']['address'] == 'ocs-ci@redhat.com'
-        assert run_data['bin_dir'] == './bin'
+        config_sections = ocsci.config.to_dict().keys()
+        for section_name in config_sections:
+            section = getattr(ocsci.config, section_name)
+            assert section == ocsci.config.get_defaults()[section_name]
+
+    def test_defaults_specific(self):
+        # FIXME: Is this test still useful?
```

Comment:
I think its still useful to check specific to verify init worked (in case aboe keys() return emtpy)

Reply:
Sure, good point - I'll just remove the FIXME there then.

Reply:
Is there a way to verify init worked without checking specific default values? I can just see updating this test in the event that we change one of these defaults being tedious.